### DISCUSSION TITLE
Profilo Genitori non obbligatorio

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#531] [Idee e proposte]
@trinko sarebbe bello non avere l'obbligatorietà di inserire l'anagrafica di entrambi o quanto meno di uno dei genitori.... Gli alunni infatti potrebbero essere maggiorenni!

**Comment by trinko** [comment]
Se usi la procedura di importazione per i nuovi utenti, non è necessario specificare i dati dei genitori: il formato del file dei è sempre quello, ma lasciando in bianco i dati dei genitori vengono creati gli utenti genitori con dati predefiniti.

**Comment by fabriziotoppi** [comment]
PARLI DEL FILE CSV?

**Comment by trinko** [comment]
> PARLI DEL FILE CSV?

Si, esatto.
